src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -581,6 +588,14 @@ export class TelemetryHelper {
     }
     public sendCodeScanEvent(languageId: string, jobId: string) {
         getLogger().debug(`start sendCodeScanEvent: jobId: "${jobId}", languageId: "${languageId}"`)
+
+        let codewhispererRuntimeLanguage: string = languageId
+        if (codewhispererRuntimeLanguage === 'jsx') {
+            codewhispererRuntimeLanguage = 'javascript'
+        } else if (codewhispererRuntimeLanguage === 'tsx') {
+            codewhispererRuntimeLanguage = 'typescript'
+        }
```

Comment:
Logic is repeated in 4 different places. Could be extracted to a function like toTelemetryLang

Reply:
@justinmk3 yea, actually I am thinking to refactor the language mapping stuff with this PR https://github.com/aws/aws-toolkit-vscode/pull/3933/files